src/tiledquick/qml/main.qml

```diff
@@ -76,13 +77,41 @@ ApplicationWindow {
         onTriggered: Qt.quit()
     }
 
+    // View
+    Action {
+        id: fitMapInViewAction
+        text: qsTr("Fit Map In View")
+        shortcut: "Ctrl+/"
+        onTriggered: {
+            // The amount that the map would need to be scaled by to fit within the view.
+            let widthRatio = mapView.width / mapItem.width
+            let heightRatio = mapView.height / mapItem.height
+            // If we need to downscale the map to fit in the view, choose the lesser ratio
+            // because that will result in the largest downscaling, which ensures the map
+            // fits both vertically and horizontally.
+            // If we need to upscale, we also want to choose the smaller ratio, as we want
+            // both to fit.
+            if (widthRatio <= heightRatio)
+                mapContainer.scale = widthRatio
+            else
+                mapContainer.scale = heightRatio
```

Comment:
Maybe we could just do `mapContainer.scale = Math.min(widthRatio, heightRatio)`. :-)

Reply:
Yep, that would be better. I initially had more more than one line in each if statement...